CI Submission wf

mickahell opened the pull request:
### Summary
Put in place the CI submission

- [ ] Get submission data
- [x] Add Tests check
- [ ] Create MR
- [ ] Put error log in issue if fail
- [ ] Add cmd issue trigger


Closes #15 
Epic #9

mickahell commented:
Everything looks up, now let's test 🚀
I didn't find how to get the log output for failure job, so I'm adding the run url to the issue

mickahell commented:
Everything who do not need Github access is working !

IceKhan13 commented:
@mickahell let's pause on tests with act. I'm not sure we need it at this point

mickahell commented:
> In general we need to document any interactions with external systems (like actions). I will also working on it in my PR :)
> 
> In general PR looks great, we just need to fix couple of things

Maybe also in the documentation / Readme, we can explain how the ecosystem work functionally and technically

mickahell commented:
> 1. Link is broken. I think we need to provide link to specific run, not to all runs.

Because you run it with act, normally the lint point a single job like : https://github.com/qiskit-community/ecosystem/actions/runs/1355329250
See : https://github.community/t/get-runs-url/16921

---
> 2. Tests results rendered incorrectly. We have open bracken in styles and coverage tests

Ok, I corrected it

---
> 3. I think tests results in message should have version of terra

Added it

---
> 4. Should we think about adding small summary of what went wrong with submission? Cuz in 2-3 days all logs from runs in actions are disappearing. Maybe have collapsible section in comment and push there short version of logs with only error messages, so we at least can tell what went wrong?

Ok these looks to make the job -> here
I used this functionality : https://docs.github.com/en/rest/reference/actions#download-workflow-run-attempt-logs

---

I'm using the vars `GITHUB_RUN_ID` and `GITHUB_JOB` who can't be used with `act`
